libs/ledger-live-common/src/families/celo/hw-getAddress.ts

```diff
@@ -1 +1,4 @@
-export { default } from "../ethereum/hw-getAddress";
+// FIXME: should not depend on another family logic
+import { resolver } from "../evm/setup";
```

Comment:
Why not create a resolver for celo if it's needed then ? Doesn't seem like too much work and pretty sure we'll forget about this FIXME